src/doc/gui_manual/Quantitative/Lineout.rst

```diff
@@ -92,6 +91,260 @@ off the **Apply operators to all plots** check box in the **Main Window**
 so that all curves do not get the same set of Lineout operator
 attributes. 
 
+There are two factors that affect how the interpolation along the line is
+performed. These are the centering of the variable and the lineout sampling
+method. There are two types of centering and two types of sampling. The
+following sections will go into detail for the four cases.
+
+Zonal variables are constant within a cell and a lineout would be expected
+to be a step function as the line moves from cell to cell.
+
+All the images associated with the examples can be generated with the
+script lineout.py.
+
+Zonal centering with sampling
+"""""""""""""""""""""""""""""
+
+In the case of sampling, the step function will become more an more apparent
+as the number of sample points increases.
+
+In the example below there are only 12 samples points and the step function
+is only somewhat apparent, since the number of sample points within a cell
+ranges between one and three.
+
+.. _lineoutexample2:
+
+.. figure:: images/rect2d_d_12_sampled.png
+
+   A zonal variable with relatively few sample points.
+
+In the example below there are 60 sample points and the step function is
+quite apparent.
+
+.. _lineoutexample3:
+
+.. figure:: images/rect2d_d_60_sampled.png
+
+   A zonal variable with a large number of sample points.
+
+Zonal centering without sampling
+""""""""""""""""""""""""""""""""
+
+In the case of non-sampling, the sample points are choosen where the line
+intersects cell boundaries, which are lines in 2D and faces in 3D. The first
+point of the line has the zonal value of the cell it is within and the
+remaining points have the value of the cell the line is about to enter.
+In this case the step function nature of the function is completely lost.
```

Comment:
'step function nature of the function' not quite sure what you mean by 'of the function', do you mean 'of the lineout' ?

Reply:
Changed "of the function" to "of the variable".